js/directives/moment-picker.js

```diff
@@ -192,26 +226,18 @@
 		function focusEvent(isMinute) {
 			cancelTimeouts();
 			$scope.minsFocused = !!isMinute;
-			$scope.hoursFocused = !isMinute;			
+			$scope.hoursFocused = !isMinute;
 		}
 
 		function changeInput(field, validator) {
```

Comment:
i checked, this method is used when you have the ^ v arrows to imcrement/decrement the value and these are not tested, not documented and definitely not used except by timmi

I'm thinking of a better implementation of these in timmi so i can remove all that untested code from lucca-ui.

if there are bugs on timmi because of it i'll rework the code to not depend on code in timmi from lucca-ui and remove it rfom lucca-ui

if it's your only concern it's fine